templates/docs/layouts/documentation.md

```diff
@@ -0,0 +1,82 @@
+---
+wrapper_template: '_layouts/docs.html'
+context:
+  title: Documentation | Layouts
+---
+
+## Documentation layout
+
+<hr>
+
+Documentation page layout can be build using Vanilla grid and common components. It consists of header with main navigation, optional hero strip (that may contain search field), grid based content area and a footer.
+
+### Structure
+
+#### Heading
+
+Heading with the main navigation is build with [navigation component](/docs/patterns/navigation#global-navigation).
+
+Style and contents of documentation main navigation should be consistent with rest of the site.
+
+Documentation pages may have an optional search box in the main navigation.
+
+#### Hero
+
+Documentation pages can have a hero area above main content area. This part of the page will usually contain a search field.
+
+Hero area is build with strip component with grid row inside. Usually it would be shallow light strip (`.p-strip--light is-shallow`), but the specific styling can be customised to match the site branding or other design requirements.
```

Comment:
```suggestion
The hero area is built with the [Strip component]https://vanillaframework.io/docs/patterns/strip) with grid row inside. Usually, it would be shallow light strip (`.p-strip--light is-shallow`), but the specific styling can be customised to match the site branding or other design requirements.
```